tests/data/articles/潮流周刊/source.md

```
@@ -0,0 +1,296 @@
+  - [第08期 - 日本大阪海游馆](/posts/08-日本大阪海游馆)
+  - [第07期 - 重庆洪崖洞](/posts/07-重庆洪崖洞)
+  - [第06期 - 白云滑雪场](/posts/06-白云滑雪场)
+  - [第05期 - 泸沽湖](/posts/05-泸沽湖)
+  - [第04期 - 东海音乐节](/posts/04-东海音乐节)
+  - [第03期 - 万石植物园](/posts/03-万石植物园)
+  - [第02期 - 太子湾](/posts/02-太子湾)
+  - [第01期 - 安吉黄昏](/posts/01-安吉黄昏)
+
+#
+
+[![潮流周刊](https://gw.alicdn.com/imgextra/i2/O1CN01m9YYjS1QBeW5DOm3I_!!6000000001938-2-tps-400-400.png)
```

markdownlint-cli2 (0.17.2)

Spaces inside link text
null

(MD039, no-space-in-links)